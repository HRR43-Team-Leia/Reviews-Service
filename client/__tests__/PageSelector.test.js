import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16.1';
import { shallow } from 'enzyme'
import PageSelector  from '../components/PageSelector.jsx';

Enzyme.configure({ adapter: new Adapter() })

describe('PageSelector', () => {
  const currPage = 1;
  const totalPages = 8;
  const jumpToPage = () => {};
  const wrapper = shallow( <PageSelector currPage={currPage} totalPages={totalPages} jumpToPage={jumpToPage} />);

  it('PageSelector should render', () => {
    expect(wrapper).toBeDefined();
  });

  it('PageSelector snapshot should match shallow render', () => {
    expect(wrapper).toMatchSnapshot()
  });
});