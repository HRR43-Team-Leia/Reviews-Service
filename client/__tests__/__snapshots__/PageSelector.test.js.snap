// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageSelector PageSelector snapshot should match shallow render 1`] = `
<div>
  <button
    className="currPage"
    type="button"
  >
    1
  </button>
  <button
    className="pages"
    onClick={[Function]}
    type="button"
  >
    2
  </button>
  <button
    className="pages"
    onClick={[Function]}
    type="button"
  >
    3
  </button>
  <button
    className="dots"
    type="button"
  >
    ...
  </button>
  <button
    className="pages"
    onClick={[Function]}
    type="button"
  >
    8
  </button>
  <button
    className="next"
    onClick={[Function]}
    type="button"
  >
    &gt;
  </button>
</div>
`;
